Research/AMAZING_DISCOVERIES.md

```
@@ -0,0 +1,729 @@
+    
+    // Clamp health
+    if (soldier->health <= 0.0f) {
+        soldier->health = 0.0f;
+        soldier->state = STATE_DEAD;
+    }
+}
+```
+
+### Health Calculation
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)